### PR TITLE
Log activity

### PR DESCRIPTION
**Pull request by brandonjank**
This is a large two prong update to the project. First, Laravel has been updated to 5.5. Second, logging functionality for all model activity has been created. See the new /log route. Composer update will take some time to update all the dependencies.

**** THIS UPDATE REQUIRES: ****
```bash
composer update
php artisan migrate
```

### CODE REVIEW

File: app/DataTables/ActivityLogDataTable.php [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            'id',
+            'log_name',
+            'description',
+            'subject_id',
+            'subject_type',	
+            'causer_id',
+            'causer_type',
+            'properties',
+            'created_at', 
+            'updated_at'
```
Comment (brec9824) [inline app/DataTables/ActivityLogDataTable.php]: Updated at and created at display the same time so we only need one. Also some of the column names should be renamed so a user could understand what the element is.

Comment (brec9824) [reply app/DataTables/ActivityLogDataTable.php]: Also the properties column only displays [object Object] rather then useful info

Comment (brandonjank) [reply app/DataTables/ActivityLogDataTable.php]: I will make the activity log "pretty" for end users in a separate request. For now, this is the bare minimum to log whats going on with our models for debugging purposes.

Comment (brec9824) [reply app/DataTables/ActivityLogDataTable.php]: Okay sounds good merging.

**Pull request merged**